Sources/NIOHTTP2/ConnectionStateMachine/FrameReceivingStates/ReceivingDataState.swift

```diff
@@ -0,0 +1,58 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
```

Comment:
this applies to all headers, I think this should be `(c) 2018-2019` and in some cases 2017-2019

Reply:
Right, but the validator rejects them. So what's the proposed change to the validator?

Reply:
@Lukasa the one in [`swift-nio`](https://github.com/apple/swift-nio/blob/master/scripts/sanity.sh#L19) has a `function replace_acceptable_years` whose job it is to replace all acceptable year markers with the string `YEARS`. I think we should do the same for the http2 one

Reply:
`2018-2019` needs to be added as an acceptable form too